### PR TITLE
Release Katello 3.7 RC2

### PR DESCRIPTION
**Pull request by jturel**

### CODE REVIEW

File: packages/katello/rubygem-katello/rubygem-katello.spec [hunk 1]
```diff
@@ -4,15 +4,15 @@
...
 %global mainver 3.7.0
 %global release 1
 
 Name:    %{?scl_prefix}rubygem-%{gem_name}
 Summary: Content and Subscription Management plugin for Foreman
 
-Version: %{mainver}%{?prever}
-Release: %{?prever:0.}%{release}%{?prever}%{?dist}
+Version: %{mainver}
+Release: %{release}%{prever}%{?dist}
```
Comment (jturel) [inline packages/katello/rubygem-katello/rubygem-katello.spec]: I had to change this b/c obal was telling me there wasn't a corresponding changelog entry for some mangled-looking version

Comment (ehelms) [reply packages/katello/rubygem-katello/rubygem-katello.spec]: This line will need the prever -- https://github.com/theforeman/foreman-packaging/pull/2756/files#diff-f7d8204599e2f8844c3a647239ed441bR186

Comment (jturel) [reply packages/katello/rubygem-katello/rubygem-katello.spec]: gets me every time

File: packages/katello/rubygem-katello/rubygem-katello.spec [hunk 2]
```diff
@@ -183,7 +183,7 @@ Documentation for %{pkg_name}.
 gem unpack %{SOURCE0}
 %{?scl:EOF}
 
-%setup -q -D -T -n  %{gem_name}-%{version}
+%setup -q -D -T -n  %{gem_name}-%{version}%{prever}
```
Comment (ehelms) [inline packages/katello/rubygem-katello/rubygem-katello.spec]: I led you astray, drop this prever addition here

Comment (jturel) [reply packages/katello/rubygem-katello/rubygem-katello.spec]: This definitely got the build further so something else probably needs to change higher up to accompany the revert?

File: packages/katello/rubygem-katello/rubygem-katello.spec [hunk 3]
```diff
@@ -237,5 +237,8 @@ cp -pa .%{gem_dir}/* \
 %doc %{gem_instdir}/README.md
 
 %changelog
+* Mon Jul 16 2018 Jonathon Turel <jturel@gmail.com> 3.7.0-1.rc2
```
Comment (ehelms) [inline packages/katello/rubygem-katello/rubygem-katello.spec]: Needs to be 3.7.0-0.1.rc2

**Pull request merged**